src/Fixture/CompanyExtensionEnabler.php

```diff
@@ -140,8 +140,8 @@ private function enableAcquiaExtensions(): void {
     }
 
     $this->output->section('Enabling company modules & themes');
-    $this->enableModules();
-    $this->enableThemes();
+    //$this->enableModules();
```

Comment:
Hack. I don't understand why, but ORCA wants to enable my SUT as a Drupal module. Why on earth would ORCA think it's a Drupal module?